SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -66,45 +71,54 @@ @interface SDLStreamingVideoLifecycleManager() <SDLVideoEncoderDelegate>
 @property (copy, nonatomic) NSArray<NSString *> *secureMakes;
 @property (copy, nonatomic) NSString *connectedVehicleMake;
 
+@property (copy, nonatomic) NSString *appName;
 @property (assign, nonatomic) CV_NULLABLE CVPixelBufferRef backgroundingPixelBuffer;
 
 @property (strong, nonatomic, nullable) CADisplayLink *displayLink;
 @property (assign, nonatomic) BOOL useDisplayLink;
 
 @property (assign, nonatomic) CMTime lastPresentationTimestamp;
 
+@property (copy, nonatomic, getter=getVideoStreamBackgroundString) NSString *videoStreamBackgroundString;
```

Comment:
Objective-C getters best practice does not use `get` nomenclature. Remove the `getter=getVideoStreamBackgroundString` and reference the method with just the `videoStreamBackgroundString `.

Reply:
Fixed